base/compiler/abstractinterpretation.jl

```diff
@@ -2075,7 +2086,7 @@ function widenreturn(@nospecialize(rt), @nospecialize(bestguess), nslots::Int, s
         local anyrefine = false
         for i in 1:length(fields)
             a = fields[i]
-            a = isvarargtype(a) ? a : widenreturn(a, bestguess, nslots, slottypes, changes)
+            a = isvarargtype(a) ? a : widenreturn(a, bestguess, nslots, slottypes, changes, interconditional_slot)
```

Comment:
Fields are prohibited from containing Conditional/InterConditional values

Reply:
Then the code that is here not is wrong, because it likes to add them.